Update RHEL client docs

lumarel opened the pull request:
# Description

This PR updates the used channels for RHEL clients, as they should be soon in the spacewalk-common-channels file (https://github.com/uyuni-project/uyuni/pull/6810).

Beside that I updated the supported client lists, as they were not synced anymore, and removed the traditional client, where I noticed it. (if I should move this to it's own PR, I can do that)

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

keichwa commented:
Separating traditional client related changes is only needed if we want to backport these to a previous suse manager version.

keichwa commented:
I guess we must wait with merging until https://github.com/uyuni-project/uyuni/pull/6810 is merged.  See description of this PR.

lumarel commented:
I had another look after I got the uyuni-docs-helper builds working now!
And also saw that the linter was unhappy so I added that as well

keichwa reviewed (approved):
Thanks for all your fixes and cleanup!  lgtm

keichwa commented:
Something seems to cause a conflict now.  Can you resolve it?

0rnela commented:
@lumarel -- can you please add the changelog for this PR as well?

lumarel commented:
@keichwa @0rnela Thank you for the poke!
I noticed that the linter cleanup basically broke asciidoc, so I removed that change again.
And yup, resolved the merge conflicts and added a changelog 👍🏻 

It would be nice if the dependent change could have the needed short look and could get merged so this could also 🙂

keichwa commented:
> @keichwa @0rnela Thank you for the poke! I noticed that the linter cleanup basically broke asciidoc, so I removed that change again. And yup, resolved the merge conflicts and added a changelog 👍🏻
> 
> It would be nice if the dependent change could have the needed short look and could get merged so this could also slightly_smiling_face

Do you mean with "dependent change" we (who?) should check https://github.com/uyuni-project/uyuni/pull/6810 ?  If that's the case the code change probably is in spacewalk-common-channels (some config file, iirc)?  This is cumbersome ;)  It would be great if QA could help.

But you are right if I see your point correctly!

lumarel commented:
@keichwa Yeah as this PR basically reflects the changes in the spacewalk-common-channels file, which is the thing that lets you easily create the preconfigured custom channels, so it could technically be merged, but the docs won't be correct as long as the uyuni-project/uyuni#6810 PR is not merged.

Tbh, it's also not correct in the docs right now 🙂

keichwa commented:
Ok, we surely can wait until everything is in place!

0rnela commented:
@keichwa @lumarel -- are we ready to merge this PR? Does it need backporting to 4.3 or 4.2 branches?

lumarel commented:
Did another check that the content actually looks like it should (built all uyuni docs)

deneb-alpha commented on .changelog:
```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Update Redhat channel names to reflect the new custom channels for easier
+  onboarding
```
```suggestion
- Update Red Hat channel names to reflect the new custom channels for easier
  onboarding
```

deneb-alpha reviewed (changes requested):
just a small typo. there's  a suggestion :)

deneb-alpha reviewed (approved):
LGTM
